test/helpers/kubectl.go

```diff
@@ -1053,8 +1086,9 @@ func (kub *Kubectl) generateCiliumYaml(options []string, filename string) error
 
 	// TODO GH-8753: Use helm rendering library instead of shelling out to
 	// helm template
+	helmTemplate := filepath.Join(kub.BasePath(), HelmTemplate)
```

Comment:
@nebril I noticed that we were using .Join but we already have a `GetFilePath` that does it for us on the kubectl type.